server/app/services/agent/container_info_mapper.rb

```diff
@@ -122,10 +122,10 @@ def container_attributes_from_docker(container, info)
       if info['Volumes']
         attributes[:volumes] = info['Volumes'].map{|k, v| [{container: k, node: v}]}
       end
+
+      attributes['networks'] = info.dig('NetworkSettings', 'Networks')
```

Comment:
Maybe room for improvement here: should we format "Networks" so that it does not look so out-of-place within container model?